CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/FileInputPane.java

```diff
@@ -93,12 +93,12 @@ public FileInputPane(final PluginParameter<FileParameterValue> parameter, final
             final List<File> files = new ArrayList<>();
             final CompletableFuture dialogFuture;
             switch (paramaterValue.getKind()) {
-                case OPEN, OPEN_OBSCURED -> dialogFuture = FileChooser.openOpenDialog(getFileChooser(parameter, "Open")).thenAccept(optionalFile -> optionalFile.ifPresent(openFile -> {
+                case OPEN, OPEN_OBSCURED -> dialogFuture = FileChooser.openOpenDialog(getFileChooser(parameter, "Open", FileExtensionConstants.TEXT)).thenAccept(optionalFile -> optionalFile.ifPresent(openFile -> {
```

Comment:
not sure I'm a fan of this one as this default then also applies to any use of the File Input Pane, notably plugin parameters
e.g. Input File parameter of Test Parameters. Probably need to find a way to include that filter only for word cloud use case